docs/ceph-csi-upgrade.md

```diff
@@ -56,11 +63,11 @@ that will not be supported in the official releases. Builds from the master
 branch can have functionality changed and even removed at any time without
 compatibility support and without prior notice.
 
-git checkout release v2.0.0 branch
+git checkout v2.1.0 tag
 
 ```bash
 [$] git clone https://github.com/ceph/ceph-csi.git
-[$] git check v2.0.0
+[$] git checkout v2.1.0
```

Comment:
the order of `checkout` and `cd` is not correct

Reply:
done